package.json

```diff
@@ -2,11 +2,17 @@
   "author": "Stefan Goessner",
   "name": "jsonpath-plus",
   "version": "5.0.2",
-  "type": "commonjs",
+  "type": "module",
```

Comment:
We still want this as `commonjs`. The standard to look at (which Webpack and Rollup plugins may orient themselves toward) is at https://nodejs.org/api/packages.html . `module` would change the default meaning of all ".js" files to mean they are modules, which they are not. But ".mjs" files are always ESM modules, so that is safe.